api/app/Models/PoolCandidate.php

```diff
@@ -1229,4 +1233,92 @@ public function scopeProcessNumber(Builder $query, ?string $processNumber): Buil
+    {
+        $this->load(['user']);
+
+        $status = $this->pool_candidate_status;
+        $decision = null;
+
+        if (in_array($status, PoolCandidateStatus::toAssessGroup())) {
+            $assessmentStatus = $this->computed_assessment_status;
+            $overallStatus = null;
+            if (isset($assessmentStatus['overallAssessmentStatus'])) {
+                $overallStatus = $assessmentStatus['overallAssessmentStatus'];
+            }
+
+            $decision = match ($overallStatus) {
+                OverallAssessmentStatus::QUALIFIED->name => FinalDecision::QUALIFIED_PENDING->name,
+                OverallAssessmentStatus::DISQUALIFIED->name => FinalDecision::DISQUALIFIED_PENDING->name,
+                default => FinalDecision::TO_ASSESS->name
+            };
+        } else {
+
+            $decisionMap = [
+                FinalDecision::DISQUALIFIED->name => [
+                    PoolCandidateStatus::SCREENED_OUT_ASSESSMENT->name,
+                    PoolCandidateStatus::SCREENED_OUT_APPLICATION->name,
+                ],
+                FinalDecision::QUALIFIED->name => [PoolCandidateStatus::QUALIFIED_AVAILABLE->name],
+                FinalDecision::QUALIFIED_PLACED->name => PoolCandidateStatus::placedGroup(),
+                FinalDecision::TO_ASSESS_REMOVED->name => [
+                    PoolCandidateStatus::SCREENED_OUT_NOT_INTERESTED->name,
+                    PoolCandidateStatus::SCREENED_OUT_NOT_RESPONSIVE->name,
+                ],
+                FinalDecision::QUALIFIED_REMOVED->name => [
+                    PoolCandidateStatus::QUALIFIED_UNAVAILABLE->name,
+                    PoolCandidateStatus::QUALIFIED_WITHDREW->name,
+                ],
+                FinalDecision::REMOVED->name => [PoolCandidateStatus::REMOVED->name],
+                FinalDecision::QUALIFIED_EXPIRED->name => [PoolCandidateStatus::EXPIRED->name],
+            ];
+
+            foreach ($decisionMap as $key => $statuses) {
+                if (in_array($status, $statuses)) {
+                    $decision = $key;
+                    break;
+                }
+            }
+        }
+
+        try {
+            $weight = match ($decision) {
+                FinalDecision::QUALIFIED->name => 10,
+                FinalDecision::QUALIFIED_PENDING->name => 20,
+                FinalDecision::TO_ASSESS->name => 30,
+                // Set aside some values for assessment steps
+                // Giving a decent buffer to increase max steps
+                FinalDecision::QUALIFIED_PLACED->name => 200,
+                FinalDecision::DISQUALIFIED_PENDING->name => 210,
+                FinalDecision::DISQUALIFIED->name => 220,
+                FinalDecision::QUALIFIED_REMOVED->name => 230,
+                FinalDecision::TO_ASSESS_REMOVED->name => 240,
+                FinalDecision::REMOVED->name => 250,
+                FinalDecision::QUALIFIED_EXPIRED->name => 260,
+                default => null
+            };
```

Comment:
Unfortunately not but it should follow what was [described in the ticket](https://github.com/GCTC-NTGC/gc-digital-talent/issues/10846#:~:text=Another%20key%20feature%20is%20that%20when%20fetching%20paginated%20candidates%2C%20they%20can%20be%20ordered%20by%20FinalDecision.%20The%20default%20ordering%20should%20be%20like%20this%3A) :+1:

Reply:
@tristan-orourke can you confirm this?

Reply:
Based on @Jerryescandon's comment [here](https://github.com/GCTC-NTGC/gc-digital-talent/issues/10846#issuecomment-2211352110), could you just move Qualified Placed above To Assess?
The theory is that this table will be used more for matching, where Assessing Candidates is less relevant, and the "kanban" screen is where assessment happens.

Reply:
> The theory is that this table will be used more for matching, where Assessing Candidates is less relevant, and the "kanban" screen is where assessment happens.

Correct. On the "Talent placement" table its just important to have a consistent easy to understand organization, there are no actual weights for the purposes of priority. At this point we are not worried with "Who should we look at first?" Since that happens on the kanban. Here we just wanna be able to find the right person using the filters and column sorting. So we just wanna have tidy statuses.

Reply:
okay, I think it should be good now. Thanks for confirming :+1: 

[update decision map, weights](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10920/commits/c6609066373c1581e18c83e33bf5cf6ebc9d0225)